### PR TITLE
play notification sound on timer end

### PR DESCRIPTION
**Pull request by lepras**

### CODE REVIEW

File: CMakeLists.txt.user [hunk 1]
```diff
@@ -0,0 +1,319 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (MagneFire) [inline CMakeLists.txt.user:1]: This file should not be part of a PR it contains local settings that are not relevant.

File: src/main.qml [hunk 2]
```diff
@@ -185,6 +193,7 @@ Application {
         triggeredOnStart: true
         onTriggered: {
             if(seconds <= 0) {
+                notificationSound.play()
```
Comment (MagneFire) [inline src/main.qml:196]: Unfortunately this won't work when the app is closed.

**Review by MagneFire — COMMENTED** [review]
Thanks for this PR.

But unfortunately it won't work the way you think it should.
The timer where you added the `notificationSound.play()` is only used as a visual indicator.
https://github.com/AsteroidOS/asteroid-timer/blob/master/src/main.qml#L165 shows what actually  happens when you start a timer: It schedules an alarm and starts the timer (that's used as the visual indicator). The use of `countdown` indicates that it is a timer and not a regular alarm.

Why these two things? Because you still want the "alarm" even after you close the application, at that point the timer (used for the visual indicator) won't be available anymore. But the scheduled alarm will be.

Once said alarm "goes off" the `alarmpresenter` is started by `systemd`: https://github.com/AsteroidOS/asteroid-alarmclock/tree/master/alarmpresenter. This means that if you want a sound to be played in a reliable manner you would need to add your changes to that part of the code. In this case you can place your `notificationSound.play()` below: https://github.com/AsteroidOS/asteroid-alarmclock/blob/master/alarmpresenter/main.qml#L175

File: src/main.qml (hunk 2 above)
Comment (lepras) [reply src/main.qml:196]: ok will add the code into the alarm presenter ... will remove these commits make a new PR

File: CMakeLists.txt.user (hunk 1 above)
Comment (lepras) [reply CMakeLists.txt.user:1]: ahh sorry didnt check git diff first.

**Pull request closed without merging**